### PR TITLE
Consider new control-plane taint

### PR DESCRIPTION
**Pull request by mogottsch**
This PR adds the new control-plane taint to occurrences of the old one.

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by mogottsch** [comment]
Not tested.

File: pkg/template/master_template.go [hunk 1]
```diff
@@ -350,7 +350,7 @@ systemd:
         {{ end -}}
         --pod-infra-container-image={{ .Images.Pause }} \
         --register-node=true \
-        --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
+        --register-with-taints=node-role.kubernetes.io/master=:NoSchedule,node-role.kubernetes.io/control-plane=:NoSchedule\
         --kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \
         --node-labels="node.kubernetes.io/master,role=master,ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
```
Comment (whites11) [inline pkg/template/master_template.go:355]: maybe want to add the control-plane label as well?

Comment (mogottsch) [reply pkg/template/master_template.go]: good catch, I've added it :+1:

File: files/conf/k8s-addons [hunk 2]
```diff
@@ -26,7 +26,6 @@ done
 SECURITY_FILES=""
 SECURITY_FILES="${SECURITY_FILES} rbac_bindings.yaml"
 SECURITY_FILES="${SECURITY_FILES} rbac_roles.yaml"
-SECURITY_FILES="${SECURITY_FILES} psp_policies.yaml"
 SECURITY_FILES="${SECURITY_FILES} psp_roles.yaml"
```
Comment (calvix) [inline files/conf/k8s-addons:29]: is there a reason why we dont remove all PSP related resources?

File: files/conf/k8s-addons [hunk 3]
```diff
@@ -26,7 +26,6 @@ done
...
 SECURITY_FILES="${SECURITY_FILES} psp_binding.yaml"
```
Comment (calvix) [inline files/conf/k8s-addons:30]: is there a reason why we dont remove all PSP related resources?

**Pull request closed without merging**